docs/source/azure/examples/addstoragefirewall.rst

```diff
@@ -0,0 +1,28 @@
+Add storage firewall
+====================
+
+.. code-block:: yaml
+
+    policies:
+        - name: add-storage-firewall
+        description: |
+            find storage accounts without open ip list and restrict them
+        resource: azure.storage
+        
+        filters:
+        - type: value
+          key: properties.networkAcls.ipRules
```

Comment:
Does it make sense to search for the accounts without firewall enabled instead of empty ipRules for this example?

Reply:
I could be just a check for default-action eq Allow, but this example explains a little bit more elaborate scenarios keeping it relatively simple.